pkg/components/init_job.go

```diff
@@ -30,6 +31,17 @@ func initJobWithNativeDriverPrologue() string {
 	return strings.Join(commands, "\n")
 }
 
+func createChangePrimaryMediumForTablesCommands(mediumName string, replicationFactor int, tables []string) string {
+	commands := []string{}
+
+	for _, table := range tables {
+		commands = append(commands, "/usr/bin/yt set "+table+"/@media/ssd_blobs  '{\"replication_factor\" = 3; \"data_parts_only\" = %false;}'")
```

Comment:
I would have leaved default replication factor, that is specifically adjusted by operator for small testing clusters, having just one or two data nodes.

Event if you want to adjust replication factor manually, I would suggest to use ```@replication_factor``` attribute, instead of ```@media```